soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -22,6 +22,10 @@ config SYS_CLOCK_TICKS_PER_SEC
 	default 4096
 	depends on STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
 
+config TICKLESS_KERNEL
```

Comment:
This should not be need as STM32_LPTIM_TIMER is declared TICKLESS_CAPABLE, and TICKLESS_KERNEL is declared in the following way:
```
config TICKLESS_KERNEL
	bool "Tickless kernel"
	default y if TICKLESS_CAPABLE
```
So the only protection needed is a check in driver to return an error if not support which is also proposed in this PR.
I think we can remove this commit safely.

Reply:
removed